cypress/support/fragments/settings/users/waiveReasons.js

```diff
@@ -10,6 +13,12 @@ const descriptionTextField = TextField({ placeholder: 'description' });
 const getRowByReason = (reason) => cy.then(() => rootSection.find(MultiColumnListCell(reason)).row());
 const getDescriptionColumnIdex = () => cy.then(() => rootSection.find(MultiColumnListHeader('Description')).index());
 
+export const getNewWaiveReason = (name, desc) => ({
+  nameReason: name ? getTestEntityValue(name) : getTestEntityValue(),
+  description: desc ? getTestEntityValue(desc) : getTestEntityValue(),
```

Comment:
too complicated, getTestEntityValue already has such ternary operator

Reply:
changed to one call of the function getTestEntityValue(param)